D/DogAPI/Versions.toml

```diff
@@ -1,2 +1,2 @@
 ["1.0.0"]
-git-tree-sha1 = "4ce09a6f533629106581683627ba990df00f9731"
+git-tree-sha1 = "7cc90eb9872302897550442f3ea0e726e901c8c8"
```

Comment:
This is not allowed.

Reply:
There is no such a repo "https://github.com/thexxiv/DogAPI.jl.git"

Reply:
It was my old account, new repo have new git hash

Reply:
So If you didn't change the hash It won't be a problem to download the package ?